tests/_support/Step/Acceptance/TargetList.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Step\Acceptance;
+
+class TargetList extends \AcceptanceTester
```

Comment:
We need a create target list and assign some contacts, accounts, leads, users to it. We may need a step object for subpanels to achieve this.